models/Item.py

```diff
@@ -42,7 +42,7 @@ def get_item(code):
         query = """
         SELECT [Code], [Name], [Risk], [Price],
             [Quantity],
-            [MinThreshold] FROM
+            [MinThreshold], [AutoPurchaseOrder] FROM
```

Comment:
Should all static methods for items be returning [AutoPurchaseOrder] in accordance with new setters within the class?